src/ui_pots.cpp

```diff
@@ -290,13 +281,22 @@ void movedPot(byte number, int value, bool isMidi) {
 				break; // RESONANCE
 
 			case 7:
-				if (!isMidi) {
-					sendCC(34, value);
-					ui_state.lastPot = 17;
-				}
-				if (voice_state.n_held_keys() >= 1) {
-					arpStepBase = value >> 2;
+				if (filterModeHeldGlobal) {
```

Comment:
After all, we're in `ui_pots.cpp` here. It feels right to me to use the `ui_state`, since we're literally in the UI here. So `#include "ui_vars.h"` here and use `ui_state.filterModeHeld` instead here. This saves you from duplicating that variable

```suggestion
				if (ui_state.filterModeHeld) {
```